src/functions/files.ts

```diff
@@ -280,6 +314,136 @@ export const reUploadImageToS3AndRemove = async (startAt: number = 0) => {
   }
 }
 
+/**
+ * @see https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#listObjects-property
+ * @param limit Sets the maximum number of keys returned in the response. The response might contain fewer keys but will never contain more.
```

Comment:
you don't need `@param` anymore since you already documented it by typescript doc